src/main/java/org/openrewrite/maven/MavenMojoProjectParser.java

```diff
@@ -306,10 +306,16 @@ private static RawRepositories buildRawRepositories(@Nullable List<Repository> r
         return rawRepositories;
     }
 
+    private Set<Path> skipOtherMavenProjects() {
+        return mavenSession.getProjects().stream()
+                .filter(o -> o != mavenSession.getCurrentProject())
+                .map(o -> o.getBasedir().toPath())
```

Comment:
Same thing here, shouldn't these paths be resolved from `baseDir` (the instance variable)?

Reply:
No, I don't think so.